aten/src/ATen/native/cpu/layer_norm_kernel.cpp

```diff
@@ -32,15 +36,28 @@ void LayerNormKernelImplInternal(
   T* mean_data = mean->data_ptr<T>();
   T* rstd_data = rstd->data_ptr<T>();
   const T c = T(1) / static_cast<T>(N);
+  constexpr int64_t VEC_SIZE = vec256::Vec256<T>::size();
```

Comment:
It will be better to name it as kVecSize.

Reply:
Updated. Thanks!